stylesheets/_component.panels.scss

```diff
@@ -7,18 +7,25 @@
     &__title {
         border: 0;
         clear: none;
-        font-size: $font-size-xlarge;
-        line-height: 32px;
+        font-size: $font-size-large * 1.1;
+        line-height: $font-size-large * 1.3;
```

Comment:
Line height needs to be 1.5 times the font-size to meet WCAG 1.4.12 Text Spacing (AA). May also need to tweak the heading icon position to ensure vertical alignment after increasing line-height.

Reply:
Doesn't need to be, just has to not break if the line height is set to 1.5.